index.html

```diff
@@ -166,57 +165,64 @@ <h6 class="client-name">2015</h6>
         <!-- ABOUT SELF SECTION -->
         <section class="about-section-container">
             <div class="about-section-card">
-                <h2 class="name-intro">
-                    About <br>
-                    Myself
-                </h2>
-                <P class="personal-info">
-                    I’m a software developer! I can help you build a product , feature or
-                    website Look through some of my work and experience! If you like
-                    what you see and have a project you need coded, don’t hestiate to contact me.
-                </P>
-                <div class="connect"><a href="#">LET'S CONNECT</a> </div>
-                <ul class="social">
-                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-                    <li><a href="#"><img src="./images/Vector.svg" alt="M icon"></a></li>
-                    <li><a href="#"><img src="./images/VectorGit.svg" alt="github icon"></a></li>
-                    <li><img src="./images/VectorImoji.svg" alt="peace imoji"></li>
-                </ul>
-                <button type="button" class="see-prjct">Get My Resume</button>
-                <div class="lang-arrow-container">
-                    <h6 class="languages-world">Languages</h6>
-                    <nav><i class="fa fa-angle-down" aria-hidden="true"></i></nav>
-                </div>
-                <!-- Javascript icon and name area -->
-                <div class="language-container">
-                    <img src="./images/Javascript.png" class="lang-img" alt="Javascript">
-                    <p class="language">Javascript</p>
-                </div>
-                <!-- HTML icon and name area -->
-                <div class="language-container">
-                    <img src="./images/Html.png" class="lang-img" alt="Html">
-                    <p class="language css-html-lng">HTML</p>
-                </div>
-                <!-- CSS icon and name area -->
-                <div class="language-container">
-                    <img src="./images/css.png" class="lang-img" alt="css">
-                    <p class="language css-html-lng">CSS</p>
+                <div class="lang-left">
+                    <h2 class="name-intro">
+                        About <br>
+                        Myself
```

Comment:
- [ ] (OPTIONAL) Same issue here. It would be great to use the correct font family and color 💯 
- Your design:
![Annotation 2022-03-24 200214](https://user-images.githubusercontent.com/85113567/159994370-bb974c6f-a91f-419e-8c33-33de0bc6f634.jpg)
- Figma design:
![Annotation 2022-03-24 200504](https://user-images.githubusercontent.com/85113567/159994391-bf35e44c-d58a-44c4-9904-ca273955442c.jpg)